runtime/src/main/java/org/corfudb/runtime/view/LayoutBuilder.java

```diff
@@ -295,7 +294,8 @@ public LayoutBuilder assignResponsiveSequencerAsPrimary(Set<String> endpoints) {
         List<String> modifiedSequencerServers = new ArrayList<>(layout.getSequencers());
         for (int i = 0; i < modifiedSequencerServers.size(); i++) {
             String sequencerServer = modifiedSequencerServers.get(i);
-            if (!endpoints.contains(sequencerServer)) {
+            if (!endpoints.contains(sequencerServer)
```

Comment:
Maybe a one line plain text description would help the maintainers to understand the condition. (e.g. a sequencer server should not be contained or unresponsive one, or similar)

Reply:
Done.